.gitignore

```diff
@@ -11,3 +11,6 @@ config.status
 configure
 libtool
 *.log
+builds/win32/*/*.sdf
+builds/win32/*/*.VC.db
+builds/win32/*/*.VC.opendb
```

Comment:
We already have .gitignore at builds/win32. I think such changes shouild be there.